domains/parthdev.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "TheInfernitex"
+    },
+    "record": {
+        "CNAME": "TheInfernitex.github.io"
```

Comment:
404 GitHub Page - Also flagged as dangerous by browser.

Reply:
My profile page shows 404 : https://github.com/TheInfernitex ?
One of my sites hosted on github pages does show up as dangerous though. I have not found the reason yet.